src/lib/plugins/commands.js

```diff
@@ -93,36 +72,103 @@ module.exports.player = function (player, serv, { version }) {
     }
   })
 
-  player.commands.add({
+  serv.commands.add({
     base: 'selector',
     info: 'Get array from selector',
     usage: '/selector <selector>',
     op: true,
     parse (str) {
       return str || false
     },
-    action (sel) {
-      const arr = serv.selectorString(sel, player.position, player.world)
-      player.chat(JSON.stringify(arr.map(a => a.id)))
+    action (sel, ctx) {
+      const arr = ctx.player ? serv.selectorString(sel, ctx.player.position, ctx.player.world) : serv.selectorString(sel)
+      if(ctx.player) ctx.player.chat(JSON.stringify(arr.map(a => a.id)))
+      else serv.log(JSON.stringify(arr.map(a => a.id)))
     }
   })
 
-  player.handleCommand = async (str) => {
-    try {
-      const res = await player.commands.use(str, player.op)
-      if (res) player.chat(serv.color.red + res)
-    } catch (err) {
-      if (err.userError) player.chat(serv.color.red + 'Error: ' + err.message)
-      else setTimeout(() => { throw err }, 0)
+  serv.commands.add({
+    base: 'help',
```

Comment:
Could you exclude only player from server help and only console from player help ?

Reply:
I think it might be ok

Reply:
excluding will reduce page length... so added player&console only